src/devices/ubisys.ts

```diff
@@ -725,6 +726,54 @@ const definitions: Definition[] = [
         },
         ota: ota.ubisys,
     },
+    {
+        // S1-R Series 2 uses the same modelId as the regular S1-R, but the energy clusters are located in endpoint 1 (instead of 4, like the regular S1-R).
+        fingerprint: [
+            {
+                manufacturerName: 'ubisys',
+                modelID: 'S1-R (5601)',
+                endpoints: [{ID: 1, profileID: 260, deviceID: 266, inputClusters: [0, 3, 4, 5, 6, 1794, 2820], outputClusters: []}],
+            },
+        ],
+        model: 'S1-R (Series 2)',
+        vendor: 'ubisys',
+        description: 'Power switch S1-R (Series 2)',
+        extend: [
+            deviceEndpoints({endpoints: {'1': 1, '2': 2, '3': 3, '232': 232}}),
+            identify(),
+            onOff({powerOnBehavior: false}),
+            electricityMeter({cluster: 'metering', configureReporting: false}),
+            commandsOnOff({endpointNames: ['2', '3']}),
+            commandsLevelCtrl({endpointNames: ['2', '3']}),
+            commandsColorCtrl({endpointNames: ['2', '3']}),
+        ],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
```

Comment:
Everything inside here is not needed because of `electricityMeter`

Reply:
I think its needed because `configureReporting` is set to `false` for `electricityMeter`. The reason is that the `currentSummDelivered` is not bindable for this device (as far i can tell for the most ubisys devices). As workaround only the `instantaneousDemand` attribute is binded manually and `currentSummDelivered` read in `onEvent`.

Reply:
I see now, I think we can improve the `ubisysOnEventReadCurrentSummDelivered` a bit, I see 2 issues with it:
- In case the Ubisys devices is sending a lot of messages, a lot of reads are done
- In case no messages are send, the value becomes stale

We can solve this by polling at a certain interval, you can use the Tuya one as an [example](https://github.com/Koenkk/zigbee-herdsman-converters/blob/cea2c45bd37d4408702e52a476dea5c2bab730d4/src/lib/tuya.ts#L165)

Reply:
Sounds good. I will take a closer look at the weekend when I have the device for testing at hand.

Reply:
I was a bit uncertain to change this for devices i do not own myself, so i added a separate method that can be used for polling.